platformsh-flex-env.php

```diff
@@ -142,3 +145,34 @@ function mapPlatformShDatabase() : string
 
     return $dbUrl;
 }
+
+function mapPlatformShMongoDatabase() : void
+{
+    // Set the MONGODB_SERVER, MONGODB_DB, MONGODB_USERNAME, and MONGODB_PASSWORD
+    // for Doctrine, if necessary.
+    // full dsn is not valid, so we need those 4 env variables.
+    // Symfony Config will be:
+    // doctrine_mongodb:
+    //     connections:
+    //         default:
+    //             server: '%env(MONGODB_SERVER)%'
+    //             options: { username: '%env(MONGODB_USERNAME)%', password: '%env(MONGODB_PASSWORD)%', authSource: '%env(MONGODB_DB)%' }
+    //     default_database: '%env(MONGODB_DB)%'
+    // see: https://symfony.com/doc/master/bundles/DoctrineMongoDBBundle/index.html
```

Comment:
This... should be in a docblock, I think, not inline comments.  

That said, it's also buried here.  This should also be replicated, with instructions on where to put that snippet, in the README (which also needs to be updated to document this mapping).

Reply:
I changed the code, did not do any doc.